app/code/Magento/InventoryIndexer/Test/Integration/Stock/IsProductInStockTest.php

```diff
@@ -0,0 +1,238 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryIndexer\Test\Integration\Stock;
+
+use Magento\Inventory\Model\CleanupReservationsInterface;
+use Magento\InventoryApi\Api\AppendReservationsInterface;
+use Magento\InventoryApi\Api\IsProductInStockInterface;
+use Magento\InventoryApi\Api\ReservationBuilderInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class IsProductInStockTest extends TestCase
```

Comment:
Looks more like `ProductIsSalableTest`.

Reply:
How I mentioned, now we don't have any service to check 'is_salable' value from index table, but it will be in IsProductInStock implemented soon in IsProductInStockTest class. So I think we should place it here now, but it will be fixed when class IsProductInStock will be changed.